src/osd/OSDMap.cc

```diff
@@ -5857,7 +5856,7 @@ void OSDMap::check_health(CephContext *cct,
       ostringstream ss;
       ss << detail.size() << " pool(s) have non-power-of-two pg_num";
       auto& d = checks->add("POOL_PG_NUM_NOT_POWER_OF_TWO", HEALTH_WARN,
-			    ss.str(), detail.size());
+			    ss.str());
```

Comment:
@neha-ojha This appears to be the offending `add()`, but I'm not sure this PR is the right place to fix it?

Reply:
ahhh, nevermind - now I see